vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -623,3 +621,9 @@ private List getContainerEnvs(config, imageName, defaultEnvVars, defaultConfig)
 
     return containerEnv
 }
+
+private void invalidateStash(def config, String prefix) {
+ String name = "${prefix}-${config.uniqueId}"
+ echo "invalidate stash ${name}"
```

Comment:
I often read "invalid stash" in the logs and am confused. Should we phrase it differently, e.g.
"Wipe out", "clear", "purge", ...